presto-main/src/main/java/com/facebook/presto/operator/TableScanOperator.java

```diff
@@ -190,6 +190,7 @@ public void finish()
         blocked.set(null);
 
         if (source != null) {
+            recordSourceRawInputStats();
```

Comment:
@pettyjamesm why is this call needed?

Reply:
It should be fine without it, but I’ve added it for consistency with ScanFilterAndProject. Happy to remove it if you’d prefer.

Reply:
@pettyjamesm I don't understand why would it be needed. If it is not, let's remove it.

Reply:
Removed.